charts/hdfs-namenode-k8s/README.md

```diff
@@ -6,8 +6,6 @@ HDFS `namenode` running inside a kubernetes cluster. See the other chart for
   Requires Kubernetes version 1.5 and beyond, because `namenode` is using
```

Comment:
This is the same duplicate paragraph I was referring to earlier. Let's remove this and move the new suggested paragraph to the top level `charts/README.md`.